www/tutorials/graphql-clients.md

```diff
@@ -1,14 +1,12 @@
 ---
 title: GraphQL Clients
-layout: ../_core/GraphQLJSLayout
 category: GraphQL.js Tutorial
-permalink: /graphql-js/graphql-clients/
-next: /graphql-js/basic-types/
+sidebar_position: 2
```

Comment:
Please list them explicitly, it's very hard to review and maintain order if it's based on this number.

Reply:
Not sure what you mean here. This order comes from how current docs have it set.

Reply:
I guess I can create https://docusaurus.io/docs/sidebar#sidebar-object

Reply:
Yes, it is exactly what I meant.